tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -424,6 +424,33 @@ public function getUserIdByUserName(string $userName): string {
 		throw new Exception(__METHOD__ . " user with name $userName not found");
 	}
 
+	/**
+	 * The method returns groupId
+	 *
+	 * @param string $groupName
+	 *
+	 * @return string
+	 * @throws Exception|GuzzleException
+	 */
+	public function getGroupIdByGroupName(string $groupName): string {
+		$response = GraphHelper::getGroup(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$this->featureContext->getAdminUsername(),
+			$this->featureContext->getAdminPassword(),
+			$groupName
+		);
+		if ($response) {
+			$data = \json_decode($response->getBody()->getContents(), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
We have this function already. Will it work here as well?
```suggestion
			$data = $this->featureContext->getJsonDecodedResponse($response)
```